HousingRepairsOnlineApi/Helpers/NotificationConfiguration/INotificationConfigurationProvider.cs

```diff
@@ -10,9 +10,8 @@ public interface INotificationConfigurationProvider
         public string ConfirmationSmsTemplateId { get; set; }
         public string ConfirmationEmailTemplateId { get; set; }
         public string InternalEmailTemplateId { get; set; }
-        public Dictionary<string, dynamic> GetPersonalisationForInternalEmailTemplate(Repair repair);
+        public Task<Dictionary<string, dynamic>> GetPersonalisationForInternalEmailTemplate(Repair repair, IRetrieveImageLinkUseCase retrieveImageLinkUseCase);
```

Comment:
Use interface for return types for this and the following two methods too
```suggestion
        public Task<IDictionary<string, dynamic>> GetPersonalisationForInternalEmailTemplate(Repair repair, IRetrieveImageLinkUseCase retrieveImageLinkUseCase);
```